docs/README.md

```diff
@@ -1,3 +1,5 @@
 # ASM Docs Samples
 
-This directory contains code samples for cloud.google.com.
\ No newline at end of file
+This directory contains code samples for cloud.google.com.
+
+To setup a GKE environment to try out ASM, follow the README.md in the `/setup` folder in this directory
```

Comment:
**Suggestion:** The folder name `setup` could be more descriptive. :)
How about using one of these names instead:
* `terraform-quickstart`
* `terraform-setup`
* `terraform-setup`
* ...

Reply:
good point - i like terraform-quickstart! Renaming!

Reply:
What about `terrafrom-managed-asm` or `terraform-asm-mcp` instead of `terraform-quickstart` to be more preseciptive on what this folder/these instructions are doing? Someone doing ABM could have the feeling they can use this but with ASM MCP in the name we clarify this a little bit more?